### DISCUSSION TITLE
Create table failing on Postgres project

### DISCUSSION THREAD

**Discussion by christianechevarria** [nocodb/nocodb#860] [Q&A] [answered]
Hi guys, I'm looking for a little help - I just set up my NocoDB instance and was able to connect successfully to a postgres instance on [Render](https://render.com) (my hosting provider) but as soon as I try to create a new table the service fails

Here are some details that might help:

1. I'm hosting my database, postgres, as a managed service on render.com that is only accessible internally via other web services on Render
1. I'm hosting NocoDB as a Docker web service and I passed the environment variable `NC_DB` in a `.env` file with an internal connection string provided by Render
1. I omitted `NC_AUTH_JWT_SECRET` (according to the docs that was okay) I can't imagine that's what happening but I'm including just in case 

The connection to the DB is successful according to NocoDB 

In case someone can make heads or tails out of this, here are my PG logs
It seems the service disconnects just as I try to create a new table
![image](https://user-images.githubusercontent.com/23618650/145853049-8df91b94-c48a-4a58-9a8a-739cb3ebc463.png)

![image](https://user-images.githubusercontent.com/23618650/145853076-b0aab951-38f5-4988-adfd-67668a32a356.png)

There's something else that I think is interesting, when I go to copy project info this is what I get back

```
Node: **v12.22.6**
Arch: **x64**
Platform: **linux**
Docker: **false**
Database: **pg**
ProjectOnRootDB: **false**
RootDB: **sqlite3**
PackageVersion: **0.83.6**
```

I'm not sure what it could be, maybe it's related to SSL?? The error message is a bit non-specific and I can't really tell what's going on by looking at the postgres logs - I left the default connection settings on

UPDATE: I deleted the project then re-connected with SSL off and I got the same issue  

UPDATE 2: After the create table fails the project stops and it is not possible to restart it

UPDATE 3: The project actually stops working immediately after connecting

![image](https://user-images.githubusercontent.com/23618650/145855312-4a755d3b-decc-45ad-ae55-7d0cc0d2a0a1.png)

Any ideas?

**Comment by o1lab** [comment]
From the browser console, do you see any errors ? or from the docker logs ?

please paste it here.

**Reply by christianechevarria** [reply]
I get a 404 on the browser, but I'm actually thinking the problem is the connection dies immediately because I just tried re-connecting and then going back to the main dashboard and the project was stopped

It's a little strange that one second it works for the initial connection and then the project is impossible to restart

**Comment by o1lab** [comment]
Please ping me on [our discord](https://discord.gg/5RgZmkW) - Im happy to help.

**Comment by o1lab** [comment, marked as answer]
Thank you, happy to see it being sorted.

For Render, issue was postgres connection URL had to use DATABASE_URL variable than NC_DB.
